sdk/core/core-amqp/test/config.spec.ts

```diff
@@ -429,4 +430,35 @@ describe("ConnectionConfig", function() {
       });
     });
   });
+
+  describe("SharedAccessSignature", () => {
```

Comment:
Can we have a test for the `fromConnectionString()` returns a credential that has getToken() method returning the exact thing that was passed as SAS token?

Reply:
Test added.